docs/content/preview/architecture/transactions/read-committed.md

```diff
@@ -27,24 +27,24 @@ The following two key semantics set apart Read Committed isolation from Repeatab
 
 ### Read restart errors
 
-In addition to the two key requirements, there is an extra YSQL-specific requirement for its read committed isolation level: ensure that external clients don't face `kReadRestart` errors that stem from clock skew which is inherent in distributed databases due to the distribution of data across more than one physical node. PostgreSQL doesn't require defining semantics around read restart errors because it is a single-node database without clock skew. When there is clock skew, the following situation can arise in a distributed database like YugabyteDB:
+In addition to the two key requirements, there is an extra YSQL-specific requirement for read committed isolation level: ensure that external clients don't face `kReadRestart` errors that stem from clock skew which is inherent in distributed databases due to the distribution of data across more than one physical node. PostgreSQL doesn't require defining semantics around read restart errors because it is a single-node database without clock skew. When there is clock skew, the following situation can arise in a distributed database like YugabyteDB:
 
-* A client starts a distributed transaction by connecting to YSQL on a node `N1` in the YugabyteDB cluster and issuing a statement which reads data from multiple shards on different physical YB-TServers in the cluster. For this issued statement, the read point which defines the snapshot of the database at which the data will be read, is picked on a YB-TServer node `M` based on the current time of that YB-TServer. Depending on the scenario, node `M` could be the same as `N1` or not, but that is not relevant to this discussion. Consider `T1` to be the chosen read time.
+* A client starts a distributed transaction by connecting to YSQL on a node `N1` in the YugabyteDB cluster and issues a statement which reads data from multiple shards on different physical YB-TServers in the cluster. For this issued statement, the read point which defines the snapshot of the database at which the data will be read, is picked on a YB-TServer node `M` based on the current time of that YB-TServer. Depending on the scenario, node `M` could be the same as `N1` or not, but that is not relevant to this discussion. Consider `T1` to be the chosen read time.
```

Comment:
incorporated the suggestion except the change from `cluster` to `universe`. Cluster seems better here since this is in the architecture section and readers might be able to related more with `cluster`.

About `.... the read point that .... of that YB-TServer` -- I couldn't find a good way to reword/ break.... Is it fine as is?